docs/docs/sourcing-from-kentico-kontent.md

```diff
@@ -207,7 +208,9 @@ export const query = graphql`
 
 When you rerun `gatsby develop`, you'll be able to see each article as a page with content pulled from Kentico Kontent. To see a list of all pages, visit `http://localhost:8000/asdf` (or any other url that generates a 404).
 
-The body copy for this article comes from a rich text element in Kentico Kontent. Links and inline linked items (e.g., embedded videos) are not resolved by default for rich text elements. If you want to resolve them, you can use the [embedded JS SDK resolution](https://github.com/Kentico/gatsby-source-kontent#embedded-js-sdk-resolution) or query the required data in structured form for resolution and create your own React components for [inline content items and components](https://github.com/Kentico/gatsby-source-kontent#content-items-and-components-in-rich-text-elements), [links](https://github.com/Kentico/gatsby-source-kontent#links-in-rich-text-elements), and [images](https://github.com/Kentico/gatsby-source-kontent#images-in-rich-text-elements).
+The body copy for this article comes from a rich text element in Kentico Kontent. Links and inline linked items (e.g., embedded videos) are not resolved by default for rich text elements. If you want to resolve them, you can query the required data in structured form for resolution and create your own React components you could use [Rich text element component](https://github.com/Kentico/kontent-gatsby-packages/tree/master/packages/gatsby-kontent-components#rich-text-element-component) that is a part of the [@kentico/gatsby-kontent-components](https://www.gatsbyjs.org/packages/@kentico/gatsby-kontent-components) package.
+
+Since the Kontent source plugin in defining the GrphQL schema for data from Kentico Kontent, you could use this schema and extend it according to your needs. There are [some examples](https://github.com/Kentico/kontent-gatsby-packages/tree/master/site#examples) what you could do in you application.
```

Comment:
```suggestion
Since the Kontent source plugin is defining the GraphQL schema for data from Kentico Kontent, you could use this schema and extend it according to your needs. There are [some examples](https://github.com/Kentico/kontent-gatsby-packages/tree/master/site#examples) of what you could do in your application.
```